### ISSUE TITLE
Live video chat

### ISSUE THREAD

**Issue by tkosamja** [RaitaroH/YouTube-DeepDark#6]
Live video chat isn't styled.
I'll try to fix, I'm just letting this here to let you know and remind myself later on.

**Comment by MechaLynx** [comment]
Mind mentioning what browser you're on, version and what userstyle extension you're using?

There are combinations that have known issues such as not being able to style iframes. I just tested on the popped out chat and it works, so if it's that, it's a known issue and, to my knowledge, a browser bug.

**Comment by tkosamja** [comment]
Hope you guys don't judge my music taste xD

![capture](https://user-images.githubusercontent.com/26143003/37869838-4be9ee38-2fb7-11e8-851d-f0002d0e63e8.PNG)

Windows 10, Firefox 59, Stylus

**Comment by MechaLynx** [comment]
Yeah this is known about Stylus on Firefox: https://github.com/openstyles/stylus/issues/248 - I remember it coming up several times and it's a planned fix but I haven't been involved with Stylus development for the last few months so I don't know whether it's on track or not. It seems they're waiting for some API call to be implement on both browsers so the codebase isn't split. Rest assured this is not due to the userstyle or because Stylus is buggy - it's just a matter of how the API provided to the extension interacts with YouTube.

I think most of the devs on Stylus are using Chrome so stuff like this might get forgotten. You might want to open an issue on Stylus's repo (don't post in the one I linked, make a new one) but you're likely to get the answer I told you about: it's known and likely won't be fixed before the browser changes happen. This could be a regression as well so up to you. You can even join the discord and ask them directly for help just in case. Whatever you feel like, just be patient.

So until this is managed on Stylus's end, I'm afraid you'll just have to pop out chat or just deal with the unstyled chat. I'm closing this issue because it isn't the userstyle's fault but if anything else relevant comes up you can still comment.

**Comment by tophf** [comment]
I thought this was fixed in Stylus 1.3.5 early March.
At least I can't repro the bug neither in FF stable nor nightly.

![clip493](https://user-images.githubusercontent.com/1310400/39133172-3dbf6e2e-471c-11e8-9d57-88787cb29450.png)

**Comment by MechaLynx** [comment]
Hmm, not sure what was going on but even though on the popped up chat and on a separate tab with just the chat styles were applied correctly, if I had DeepDark Classic _and_ DeepDark Material (the latter is intended to style the live chat) then the embedded chat doesn't get styled at all. I've got a 3rd personal style for the chat that wasn't applied either in this case.

However, if I disable DeepDark Classic, both of the other styles (Material and my personal one) are applied as normal. When I wrote the chat styling for Material I tested with both active, on the embedded chat, the popped out chat and on a separate tab and they all worked.

I'll modify the Classic style to exclude live chat since that works. If you want to try and reproduce just in case this is a Stylus bug, I'll link the commit where the issue is reproducible here after I make the change and test it.
